src/cloud/scriptmgr/scriptmgrpb/service.proto

```diff
@@ -36,6 +36,9 @@ service ScriptMgrService {
   rpc GetScripts(GetScriptsReq) returns (GetScriptsResp);
   // GetScriptContents returns the pxl string of the script.
   rpc GetScriptContents(GetScriptContentsReq) returns (GetScriptContentsResp);
+  // Checks to see if script with hash exists. This is used to check if a script
+  // has been modified or not.
+  rpc GetScriptByHash(GetScriptByHashReq) returns (GetScriptByHashResp);
```

Comment:
The naming of this function makes me think that you'll be able to get the contents of the script given the hash, or atleast a script ID or something. You could make this more specific... Maybe `CheckScriptExists`?